src/payment/apple-pay/ApplePayButton.jsx

```diff
@@ -14,6 +15,8 @@ export default class ApplePayButton extends React.Component {
   }
 
   handleClick = () => {
+    if (this.props.disabled || this.props.totalAmount === undefined) return;
```

Comment:
Can this be different from `canMakePayments` below?

Reply:
canMakePayments is the browser capability. May be worth combining these into a single boolean.

Reply:
Seems like a helper that combines all of this would help avoid errors, but I'm not 100% sure.  I'd recommend doing something, even if in a follow-up PR.  Thanks.

Reply:
👍